components/board/digital_interrupts.go

```diff
@@ -18,6 +18,19 @@ type Tick struct {
 
 // A DigitalInterrupt represents a configured interrupt on the board that
 // when interrupted, calls the added callbacks.
+//
+// Value example:
+//
+//	myBoard, err := board.FromRobot(robot, "my_board")
+//
+//	// Get the DigitalInterrupt "my_example_digital_interrupt".
+//	interrupt, err := myBoard.DigitalInterruptByName("my_example_digital_interrupt")
+//
+//	// If using type basic: Get the amount of times this DigitalInterrupt has been interrupted with a tick.
+//	count, err := interrupt.Value(context.Background(), nil)
```

Comment:
Type basic is a pi-only implementation. This is just how many times a digital interrupt has ticked.